server/routes/popRoutes.ts

```diff
@@ -13,62 +12,12 @@ interface MessageItem {
   sender: string
 }
 
-export default function routes(): Router {
+export default function popRoutes(): Router {
```

Comment:
Weirdly, this function name doesn't need to match the import alias because it's the default export.